examples/tutorials/02_build_pipelines.ipynb

```diff
@@ -257,64 +308,150 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "Given these results, we might be interested in the relationship between height and weight *controlling for gender*. That is, given the same gender, how is change in height associated with change in weight?"
+    "# Map each species to a color\n",
+    "color_map = {\"Setosa\": \"green\", \"Versicolor\": \"blue\", \"Virginica\": \"red\"}\n",
```

Comment:
cant find the word here but since we are doing "session" instead of "day", change references to "tomorrow" in the markdown

Reply:
nvmd.. its a narrative. doesnt matter.